### PR TITLE
Fix pre-commit errors

### PR DESCRIPTION
**Pull request by timmens**

### CODE REVIEW

File: src/labor_supply_norms/sandbox/questions/applied_micro_coffee_14_04_23.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Questions
```
Comment (ChristianZimpelmann) [inline src/labor_supply_norms/sandbox/questions/applied_micro_coffee_14_04_23.md:1]: I think the questions would rather fit in this repo: https://gitlab.iame.uni-bonn.de/ChristianZimpelmann/labor_supply_norms_admin

File: src/labor_supply_norms/data_management/task_impute_norms_ind_level.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def task_impute_values(depends_on, produces, prediction_specs=PREDICTION_SPECS):
             data = create_norm_groups(data, var)
 
     # Create groups for orginal variable
-    for var in ["norms_gender_cons_avg"]:
+    for var in "norms_gender_cons_avg":
```
Comment (ChristianZimpelmann) [inline src/labor_supply_norms/data_management/task_impute_norms_ind_level.py:78]: round brackets missing?

**Review by ChristianZimpelmann — CHANGES_REQUESTED** [review]
Thanks! But I thought the feedback and questions rather fit to the admin repo. We can discuss if you disagree of course

**Comment by timmens** [comment]
Yes, thank you for mentioning it again. I've opened a PR in the correct repo where I added the feedback.

I think we can use this PR to fix the few pre-commit errors. I do not necessarily like the `(1,)` syntax for a tuple with only one element, so I'd also be fine with ignoring ruff's rule against looping over `[1]`; especially since I think the latter is more readable.

**Comment by ChristianZimpelmann** [comment]
Thanks for fixing the pre-commit errors. 

> I think we can use this PR to fix the few pre-commit errors. I do not necessarily like the `(1,)` syntax for a tuple with only one element, so I'd also be fine with ignoring ruff's rule against looping over `[1]`; especially since I think the latter is more readable. 

So you would deactivate the rule converting those lists to sets in general or only for collections of one element only?
I don't mind changing lists to sets, but I agree that the comma after just one element is strange.

So I tend to prefer `(1)`. But don't have a strong opinion here.

**Review by ChristianZimpelmann — APPROVED** [review]
Feel free to implement your preferred changes.

**Comment by timmens** [comment]
Except for the few cases I changed here, all other collections you iterate over are already tuples. I decided to stick with the tuple and the additional comma because it serves as a good reminder that we should either add more variables to the collection at some point or not loop over it at all.

**Pull request merged**